### DISCUSSION TITLE
PROJECT: Short Term Service Experiments

### DISCUSSION THREAD

**Discussion by tech-ea** [excelerate-america/ea-features#275] [General]
# Short Term Service Experiments

#### Conversion Method: Paid Ads
- Paid Ad to main biz essential landing page and unique promoCode (Normal Route)
- Paid Ad to new custom onboard pathway (Same Day printing) and unique promoCode
- Paid Ad to new custom onboard pathway (Pricing Hot Items)  and unique promoCode
#### Conversion Method: Direct Outreach
- New custom onboard pathway and unique promoCode for calls 
- New custom onboard pathway unique promoCode via chat (see below)

#### Conversion Method: Community Partner Outreach
- Email to community partner lists directing link to community partner/biz esssentials start page
- Ability to create a list of member signups from promo group
- Start this by 11/28 at the latest

### 2. Can we convert SBC Card downloaders from webflow  into biz essentials?
- SBC Card Pathways currently webflow form --> google sheet and sendgrid trigger
- Once we demonstrate some traction here we should
- Create a page/form for lead generation to target on ea.com
- Create tracking of conversion on heap
- Get lead form data into zendesk and trigger sendgrid send of SPC card

### 3. Do people want more than ODP/biz essentials?
> Do this by --> Creating a follow up survey to send to new conversion asking what else are they interested in.  Could be just typeform for now.  Additionally, consider creating a section on "service page" we discussed that has "coming soon" sections that a public user or member can show interest in.

### To test 3, we need to create a simple way to initiate communication between ea and business account holders from within  the site or connected to the site
- Setup a 'Members Only' page that can contain chat, surveys, and  static, dynamic, and curated content to test other possible services
- Add and Test ZendDesk Chat/ messaging 
- Investigate TypeForm integration
- Investigate SurveyJS or similar service to create and deploy surveys from inside site

## TERMS
special vocabulary 
## QUESTIONS

- Should it be 3 custom onboard pathways? or modified Start and confirm page based on promoCode
## TASKS
- [ ] Support Grant Competition Updates
- [ ] Determine Contacts , Subscribers, Members, Businesses single source of truth
- [ ] Determine Tracking Contacts , Subscribers, Members, Businesses single source of truth
- [ ] Support Content Team on Paid Ad to main biz essential landing page and unique promoCode (Normal Route)
- [ ] Support Content Team on Paid Ad to new custom onboard pathway (Same Day - [ ] printing) and unique promoCode
- [ ] Support Content Team on Paid Ad to new custom onboard pathway (Pricing Hot - [ ] Items) and unique promoCode
- [ ] New custom onboard pathway and unique promoCode for calls
- [ ] New custom onboard pathway unique promoCode via chat (see below)
- [ ] Support Content Team on SBC Card downloaders from webflow into biz essentials?
- [ ] Add and Test ZendDesk Chat/ messaging
- [ ] Setup a 'Members Only' page that can contain chat, surveys, and static, dynamic, and curated content to test other possible services
- [ ] Investigate TypeForm integration
- [ ] Investigate SurveyJS or similar service to create and deploy surveys from inside site
Add an item

**Comment by tech-ea** [comment]
@davidcjanssens Here is a project issue to  start refining and adding tasks to. I tried to document what you said with some notes i took. edit if necessary

**Comment by davidcjanssens** [comment]
Adding some details to conversions and SBC

**Comment by tech-ea** [comment]
@sorengrey

**Comment by sorengrey** [comment]
**Some notes from yesterday's Design Review Meeting:**

- We will be introducing **Guides**, a new type of slug page that will operate almost exactly like Reads.
- The Figma for Guide slug and index pages, including mobile view, [can be found here. ](https://www.figma.com/file/5y261frxJtwW1NgsPcq3gi/Testing-Flows?node-id=1289%3A7391)
- The third section (titled "Guides"), will not be published to the Slug Active page until there are 3 published Guides in HyGraph
- We will need to create a new card for the fifth section (titled "Shopping Shortcuts") that will feature an image, a title, a new LayerStyle, and button. 
- We will also need a new card for the last section (titled "Quick tutorials"), that will feature a headline, a subheadline, and a video hosted by **Loom**
- The content on Guide slug pages will be based off of material in ODP's [User Guide](https://media.officedepot.com/image/upload/v1652207327/content/Web_-_Complete_User_Guide.pdf).
